apptools/preferences/tests/test_preferences_helper.py

```diff
@@ -92,7 +92,7 @@ class AcmeUIPreferencesHelper(PreferencesHelper):
         self.assertEqual('blue', helper.bgcolor)
         self.assertEqual(50, helper.width)
         self.assertEqual(1.0, helper.ratio)
-        self.assertEqual(True, helper.visible)
+        self.assertTrue(helper.visible)
```

Comment:
I could not find an assertion in these tests that check `assertFalse(helper.visible)`.  `helper.visible` could be a method instead of a scalar value, and that would pass this `assertTrue` check.
```
>>> class A:
...     def visible(self):
...         return False
... 
>>> bool(A().visible)
True
```

Reply:
Should I revert this change?  In all the `AcmeUIPreferencesHelper` class definitions, `visible` is specified as a `Bool`

Reply:
Sorry. Never mind this one then.